exporter/keys.go

```diff
@@ -235,7 +202,55 @@ func parseKeyArg(keysArgString string) (keys []dbKeyPair, err error) {
 			return keys, fmt.Errorf("couldn't parse db/key string: %s", k)
 		}
 
+		// We want to guarantee at the top level that invalid values
+		// will not fall into the final Redis call.
+		if db == "" && key == "" {
```

Comment:
I think you can simplify this and just check for `empty or empty` - the addtl granularity in the error message isn't really needed.